forge-core/src/main/java/forge/deck/DeckRecognizer.java

```diff
@@ -414,7 +414,7 @@ public static TokenKey fromString(String keyString){
     public static final String REGRP_CARD = "cardname";
     public static final String REGRP_CARDNO = "count";
 
-    public static final String REX_CARD_NAME = String.format("(\\[)?(?<%s>[a-zA-Z0-9&',\\.:!\\+\\\"\\/\\-\\s]+)(\\])?", REGRP_CARD);
+    public static final String REX_CARD_NAME = String.format("(\\[)?(?<%s>[a-zA-Z0-9âûáóéíúÉàäöüñ&',\\.:!\\+\\\"\\/\\-\\s]+)(\\])?", REGRP_CARD);
```

Comment:
Would something like À-ÿ work here to cover all bases?

Reply:
i only added the ones i found, i don't know how the letter ranges works for such ones

i thought about using Unicode for the Regexp, but i thought that might be overkill

Reply:
As far as I know it should include everything we'd need while future proofing. I know it finds all cards on Scryfall so might be worth it if it works in Forge

https://scryfall.com/search?q=name%3A%2F%28%5B%C3%80-%C3%BF%5D%29%2F+-t%3Acard

Reply:
i think the only ones missing are some un cards, but they should not matter